notes/dpla_map_and_pcdm.md

```diff
@@ -0,0 +1,50 @@
+
+## Relationship between DPLA MAP and PCDM
+
+This document notes the ways in which the DPLA Metadata Application Profile and PCDM can be used together.
+
+### Mapping from PCDM to DPLA MAP
+
+#### Collections
+
+One or more `pcdm:Collection`s map to a `dcmitypes:Collection`. As `pcdm:Collection`s may be hierarchical, and DPLA's use of `dcmitypes:Collection` is not hierarchical, information may be accumulated from multiple `pcdm:Collection`s into one.  
```

Comment:
Zero or more collections map? How would you map from no collection to a collection?  ... or ... you can certainly create collections from external information, but that's not the point of this doc?